src/Depth.sol

```diff
@@ -43,8 +43,31 @@ contract Depth is IDepth {
                 PoolTickBitmap.nextInitializedTickWithinOneWord(poolVariables, upper ? tick : tick, !upper);
```

Comment:
`nextInitializedTickWithinOneWord` and `_findNextTick` shouldn't have to have another param for `tick` bc its included in poolVariables